src/Components/ImagesTable/ImagesTable.tsx

```diff
@@ -73,6 +74,17 @@ type ImageTableProps = {
 const ImagesTable = ({ selectedBlueprint }: ImageTableProps) => {
   const [page, setPage] = useState(1);
   const [perPage, setPerPage] = useState(10);
+  const { selectedBlueprintVersion } = useGetBlueprintsQuery(
+    {},
+    {
+      selectFromResult: ({ data }) => ({
+        selectedBlueprintVersion: data?.data?.find(
+          (blueprint: { id: string | undefined }) =>
```

Comment:
Let's use `BlueprintItem` (exported from `imageBuilderApi.ts`) as the type here instead of `{ id: string | undefined }`.

Reply:
Force pushed these changes to your branch myself so we can hopefully just get this in, hope that's okay with you. :sweat_smile: